workflow/rules/maftools.smk

```diff
@@ -0,0 +1,36 @@
+rule group_bcf_to_vcf:
+    input:
+        "results/final-calls/{group}.{event}.{calling_type}.fdr-controlled.bcf",
+    output:
+        temp("results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf"),
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    params:
+        extra="",
+    wrapper:
+        "v3.8.0/bio/bcftools/view"
+
+
+rule group_vcf_to_maf:
+    input:
+        vcf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf",
+        ref=genome,
+    output:
+        maf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.maf",
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    conda:
+        "../envs/vcf2maf.yaml"
+    params:
+        genome_build=config["ref"]["build"],
+        species=config["ref"]["species"],
+    shell:
+        "vcf2maf.pl --inhibit-vep "
+        " --retain-info PROB_ARTIFACT,PROB_ABSENT,PROB_SOMATIC,PROB_GERMLINE_HOM,PROB_GERMLINE_HET "
+        " --retain-fmt DP,AF "
+        " --ncbi-build {params.genome_build} "
+        " --species {params.species} "
+        " --vcf-tumor-id tumor --vcf-normal-id panel_of_normals "
+        " --input-vcf {input.vcf} --output-maf {output.maf} --ref-fasta {input.ref} "
```

Comment:
Does maftools read the input vcf multiple times? If not, I suggest to just use an anonymous pipe here and get rid of the bcftools rule above: `<(bcftools view input.bcf)`.

Reply:
Will this work with a commandline option that expects a path?

Reply:
Yes, that works. As long as it does not open the path multiple times.

Reply:
It seems to be doing some non-standard kind of reading, and the construct `--input-vcf <(bcftools view {input.bcf})` does not work, but throws this:
```
===========================================================
ERROR: Provided --input-vcf is missing or empty: /dev/fd/63
===========================================================
```
A regular `bcftools view <filename>` with the same file in the same conda environment does work. So I'm assuming this is an issue of vcf2maf input reading...